configuration.k

```diff
@@ -135,10 +135,10 @@ imports SOLIDITY-SYNTAX
                         <ctLocation> .Map </ctLocation>
                         <tempContext> .Map </tempContext>
                         <tempStore> .Map </tempStore>
-                        <tempdistore> .Map </tempdistore>
-                        <tempmemory> .Map </tempmemory>
-                        <ditype> .Map </ditype>
-                        <distore> .Map </distore>
+                        <tempIdstore> .Map </tempIdstore>
+                        <tempMemory> .Map </tempMemory>
+                        <idtype> .Map </idtype>
```

Comment:
should be idType? (following same convention as other names)

Reply:
dstore is dynamic store to store the content of dynamic array.
Idstore is the dynamic store in the contract instance.

Reply:
my comment was just about the capital letter - I just thought it should be `idType` instead of `idtype` since you have been using the same convention elsewhere (and it's a bit easier to read, too).